ZuluContent/Spells/Sixth/Dispel.cs

```diff
@@ -1,77 +1,39 @@
+using System.Threading.Tasks;
 using Server.Items;
 using Server.Mobiles;
 using Server.Targeting;
 
 namespace Server.Spells.Sixth
 {
-    public class DispelSpell : MagerySpell
+    public class DispelSpell : MagerySpell, ITargetableAsyncSpell<Mobile>
     {
-        public DispelSpell(Mobile caster, Item spellItem) : base(caster, spellItem)
-        {
-        }
+        public DispelSpell(Mobile caster, Item spellItem) : base(caster, spellItem) { }
         
-        public void Target(Mobile attacker, Mobile defender)
+        public async Task OnTargetAsync(ITargetResponse<Mobile> response)
         {
+            if (!response.HasValue)
+                return;
 
-            var bc = defender as BaseCreature;
+            var target = response.Target;
+            
+            SpellHelper.Turn(Caster, target);
+            //t.IsDefined(typeof(DispellableFieldAttribute), false)
```

Comment:
Could that be removed?

Reply:
Nah that's a note to myself, Dispel is a WIP at the moment, I implemented the bare functionality to dispel players to test buffs.

Dispel also needs to be able to delete a single summoned field, e.g. only removing one block of Wall of Stone/Fire Field which is what I need that commented line for.